daemon/api.go

```diff
@@ -661,6 +661,7 @@ func postSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 
 	chg := newChange(state, inst.Action+"-snap", msg, tsets)
 	chg.Set("snap-names", []string{inst.snap})
+	chg.Set("api-data", map[string]string{"snap-name": inst.snap})
```

Comment:
Hmm.. I now realize the first one is not even on api-data. Why do we need that?  Looking...